### DISCUSSION TITLE
Implement Windows Music Bar

### DISCUSSION THREAD

**Discussion by Layendan** [ppy/osu#13244] [Ideas] [answered]
I know the title might not make a lot of sense, but the idea is to utilize Windows Built-In music bar:
![Windows Music Bar](https://user-images.githubusercontent.com/46062388/120093932-42b8bb00-c0d2-11eb-8033-e568f7f4090a.png)
![image](https://user-images.githubusercontent.com/46062388/120093999-9cb98080-c0d2-11eb-951a-db8e6f0de175.png)

I was thinking of turning on this while on the MainMenu page if not maybe on the PlaySongSelect page:
![image](https://user-images.githubusercontent.com/46062388/120094028-c8d50180-c0d2-11eb-80f6-9c7641bc6dd6.png "MainMenu")
![image](https://user-images.githubusercontent.com/46062388/120094086-2b2e0200-c0d3-11eb-83b9-c290dab16ce5.png "PlaySongSelect")
This is standard for most music players and I think that it would be useful for people who use Osu! for music.

**Comment by frenzibyte** [comment]
I'm not sure this will happen, it is a standard for music players, but osu!lazer is not a music player.

**Comment by gagahpangeran** [comment]
I dont think this should be implemented. As I know, peppy is very against using osu! as music player.

**Comment by peppy** [comment, marked as answer]
Out of scope, won't happen.
